CHANGELOG.rst

```diff
@@ -20,6 +20,11 @@ Changed
   official instructions at https://docs.mongodb.com/v3.4/release-notes/3.4-upgrade-standalone/.
   (improvement)
 
+Deprecated
+~~~~~~~~~~
```

Comment:
But then again, maybe it still belongs in "Deprecated" since we're making it hard to use, but not impossible, not yet anyway